lib/talkable/generators/templates/app/views/invite/show_offer.html.slim

```diff
@@ -0,0 +1,2 @@
+#talkable-inline-offer-container
+= render 'shared/talkable_offer', offer: @invite_offer, options: {iframe: {container: 'talkable-inline-offer-container'}}
```

Comment:
I am not sure why this partial is need. It is used to display an offer quite differently in floating widget, PP and Invite from the UI stand point and I hardly imagine someone would want to reuse a peace of code in between.